docs/pages/includes/database-access/auto-discovery-tip.mdx

```diff
@@ -0,0 +1,4 @@
+This guide shows how to register a single {{ dbType }} with your
+Teleport cluster. For a more scalable approach, learn how to setup
```

Comment:
```suggestion
Teleport cluster. For a more scalable approach, learn how to set up
```
"Setup" is the noun form. "Set up" works similarly to "lie down," "stand up", etc.